### PR TITLE
Added about button (#84)

### PR DESCRIPTION
**Pull request by CharlesGameDev**
Added an about button that shows the current Scarab version and the GitHub link.
Answers issue #84

### CODE REVIEW

**Review by Cavernosa — CHANGES_REQUESTED** [review]

File: Scarab/ViewModels/ModListViewModel.cs [hunk 1]
```diff
@@ -137,6 +140,15 @@ private async Task ChangePathAsync()
             (Application.Current?.ApplicationLifetime as IClassicDesktopStyleApplicationLifetime)?.Shutdown();
         }
 
+        private async Task AboutAsync()
+        {
+            string aboutText = Resources.PU_About;
```
Comment (fifty-six) [inline Scarab/ViewModels/ModListViewModel.cs:145]: Just method chaining would be nicer here

File: Scarab/ViewModels/ModListViewModel.cs [hunk 2]
```diff
@@ -137,6 +140,15 @@ private async Task ChangePathAsync()
...
+            aboutText = aboutText.Replace("1", Assembly.GetEntryAssembly().GetName().Version.ToString());
```
Comment (fifty-six) [inline Scarab/ViewModels/ModListViewModel.cs:146]: It's not gonna be relevant so you can suppress the null warning here with `!`. Alternatively `?` would be fine (maybe preferred, given I was gonna look into AOT stuff) and just default to "Unknown" with `??`.

**Review by fifty-six — CHANGES_REQUESTED** [review]

**Comment by fifty-six** [comment]
What Cavernosa said also seems good to address, aside from that this looks great though.

**Comment by fifty-six** [comment]
Went with a tab in the latest release instead, though thanks for the PR

**Pull request closed without merging**